enterprise/migrations/0172_auto_20230512_2232.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.19 on 2023-05-12 22:32
+
+from django.db import migrations, models
+
+from enterprise.models import EnterpriseCourseEnrollment
+
+
+def backfill_unenrolled_enterprise_enrollments(apps, schema_editor):
+    enterprise_enrollments = EnterpriseCourseEnrollment.objects.all()
```

Comment:
[context] we have to import this table directly in order to get access to the `course_enrollment` property